docs/content/develop/resource.md

```diff
@@ -0,0 +1,455 @@
+   # create_verb: :POST
+
+   # Overrides the update URL for the resource. (Otherwise, the self_link URL
+   # will be used.)
+   # Terraform field names enclosed in double curly braces will be replaced
+   # with the field values from the resource.
+   # update_url: 'projects/{{project}}/locations/{{location}}/resourcenames/{{resource_id}}'
+   # The HTTP verb used to update a resource. Allowed values: :POST, :PUT, :PATCH. Default: :PUT.
+   update_verb: :PATCH
+   # True if the resource should use an update mask for updates.
+   update_mask: true
+
+   # Overrides the delete URL for the resource. (Otherwise, the self_link URL
+   # will be used.)
+   # Terraform field names enclosed in double curly braces will be replaced
+   # with the field values from the resource.
+   # delete_url: 'projects/{{project}}/locations/{{location}}/resourcenames/{{resource_id}}'
+   # Overrides the HTTP verb used to delete a resource.
+   # Allowed values: :POST, :PUT, :PATCH, :DELETE. Default: :DELETE
+   # delete_verb: :DELETE
+
+   # Enables generation of code to handle API calls that return operations.
+   autogen_async: true
+   # Sets parameters for handling operations returned by the API.
+   async: !ruby/object:Api::OpAsync
+     # Overrides which API calls return operations. Default: ['create',
+     # 'update', 'delete']
+     # actions: ['create', 'update', 'delete']
+     operation: !ruby/object:Api::OpAsync::Operation
+       base_url: '{{op_id}}'
+     # If true, the completed operation's returned JSON will be expected to
+     # contain a full resource in the "response" field
+     # result: !ruby/object:Api::OpAsync::Result
+     #   resource_inside_response: true
+
+   # All resources (of all kinds) that share a mutex value will block rather
+   # than executing concurrent API requests.
+   # Terraform field names enclosed in double curly braces will be replaced
+   # with the field values from the resource.
+   # mutex: RESOURCE_NAME/{{resource_id}}
+
+   # IAM_GOES_HERE
+
+   # EXAMPLES_GO_HERE
+
+   parameters:
+     - !ruby/object:Api::Type::String
+       name: 'location'
+       required: true
+       immutable: true
+       url_param_only: true
+       description: |
+         LOCATION_DESCRIPTION
+     - !ruby/object:Api::Type::String
+       name: 'resource_id'
+       required: true
+       immutable: true
+       url_param_only: true
+       description: |
+         RESOURCE_ID_DESCRIPTION
+
+   properties:
+     # Fields go here
+   ```
+
+3. Modify the template as needed to match the API resource's documented behavior.
+
+> **Note:** The template includes the most commonly-used fields. For a comprehensive reference, see [ResourceName.yaml reference ↗]({{<ref "/reference/resource-reference.md" >}}).
+{{< /tab >}}
+{{< tab "Handwritten" >}}
+> **Warning:** Handwritten resources are much more difficult to develop and maintain. New handwritten resources will only be accepted if the resource cannot be implemented using MMv1.
```

Comment:
Ehh- most of this statement isn't really true. They're harder to maintain, around equally difficult to develop, and would be accepted in cases where MMv1 implementation is possible but difficult.

Reply:
softened language in 42a6bbffa

Reply:
nit: Can we use "Two or more" or "Multiple"? I've personally found "More than one" to introduce confusion by grounding folks on the "one" part.

Reply:
yeah, I agrre that's clearer, thanks! changed in 177b89197